lib/modules/platform/scm-manager/schema.ts

```diff
@@ -0,0 +1,121 @@
+import { z } from 'zod';
+
+export const UserSchema = z.object({
+  mail: z.string().optional().nullable(),
+  displayName: z.string(),
+  name: z.string(),
+});
+export type User = z.infer<typeof UserSchema>;
```

Comment:
```suggestion
export const User = z.object({
  mail: z.string().optional().nullable(),
  displayName: z.string(),
  name: z.string(),
});
export type User = z.infer<typeof User>;
```

use same name, typescript is smart enough to distinguish.